docs/user-guide/rules/cloudinary.md

```diff
@@ -0,0 +1,79 @@
+# Image optimization with Cloudinary (`cloudinary`)
```

Comment:
I think having the rule just named `cloudinary` may be a bit confusing. Maybe have image optimization in the name of the rule?

Reply:
Image-optimization-cloudinary sounds de good?

Reply:
> Image-optimization-cloudinary sounds de good?

Yes.

Cc: @sonarwhal/core?